Source/Diagnostics/FlushFormats/FlushFormatSensei.cpp

```diff
@@ -70,6 +70,16 @@ FlushFormatSensei::WriteToFile (
     amrex::ignore_unused(varnames, mf, iteration, time, particle_diags);
 #else
     WARPX_PROFILE("FlushFormatSensei::WriteToFile()");
```

Comment:
Please see BTD comment & need for assert above.

Reply:
Ok, I added an assert instead